frontend/src/components/PayMethod.vue

```diff
@@ -72,12 +122,60 @@ export default {
     navbar,
     foot
   },
+  computed: {
+    cardState () {
+      if (this.card_number.toString().length === 16 && this.card_number >= 1000000000000) {
+        return true
+      }
+      return false
+    },
+    cardInvalid () {
+      if (this.card_number.toString().length === 16 && this.card_number < 1000000000000) {
+        return 'Introduzca el número de tarjeta valido'
+      }
+    },
+    nameState () {
+      return this.card_holder_name.length > 0
+    },
+    nameInvalid () {
+      if (this.card_holder_name.length <= 0) {
```

Comment:
Cambiaria está condición para ser coherente con la de nameState
```suggestion
      if (this.card_holder_name.length < 0) {
```